decidim-core/app/views/layouts/decidim/participatory_process.html.erb

```diff
@@ -1,46 +1,55 @@
 <%= render "layouts/decidim/application" do %>
-<main class="wrapper">
-  <div class="process-header process-header--home row column">
-    <div class="row column process-header__main"
-      style="background-image:url(<%= current_participatory_process.banner_image.url %>);">
-      <div class="process-header__container row">
-        <div class="columns mediumlarge-9 process-header__info">
-          <h1 class="text-highlight heading1">
-            <%= translated_attribute(current_participatory_process.title) %>
-          </h1>
-          <h2 class="text-highlight heading4">
-            <span class="process-header__hashtag">
-              <%= current_participatory_process.hashtag %>
-            </span>
-            <%= translated_attribute(current_participatory_process.subtitle) %>
-          </h2>
+  <div class="wrapper">
```

Comment:
@beagleknight the <main> tag is just for semantic purposes, but the important part is the class. It can be unified to a <div> if it makes things easier.

The footer-separator is kind of a wrapper to separate the content from the footer and make a sticky footer with flexbox.